cluster/gce/gci/master.yaml

```diff
@@ -1,6 +1,20 @@
 #cloud-config
 
 write_files:
+  - path: /etc/systemd/system/kube-protect-systemd.service
```

Comment:
Have you thought of doing it in [configure-helper.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh)?
Alternatively, here is fine too. I'd suggest though run it as a 'bootcmd' instead.

bootcmd:
 - echo -999 > /proc/`pidof systemd`/oom_score_adj